tools/sigh.ts

```diff
@@ -363,6 +363,9 @@ function buildPath(path: string, preprocess: () => void, deps: string[]): () =>
 
     return true;
   };
+  // Using a lambda breaks the display of func.name in the main execution loop.
+  Object.defineProperty(fn, 'name', {value: 'build'});
```

Comment:
Seem like an opportunity to show the path that is being built with ``` `building ${path}` ```?

Reply:
Done.